pbs/pbsrequest.go

```diff
@@ -229,8 +230,8 @@ func (req PBSRequest) Elapsed() int {
 	return int(time.Since(req.Start) / 1000000)
 }
 
-func (req PBSRequest) GetUserID(BidderCode string) string {
-	if uid, ok := req.UserIDs[BidderCode]; ok {
+func (req PBSRequest) GetUserID(BidderCode family.Name) string {
+	if uid, ok := req.UserIDs[BidderCode.String()]; ok { // TODO: consider moving UserIDs to an array indexed by the FamilyName int.
```

Comment:
It was either dig in here to make sure I caught every usage of UserIDs, or put it off for a later PR. Seems a bit more manageable in chunks rather than starting off with a huge PR as my first.

The comment above about "family" makes sense and I will work on it, and I will take a look at how much UserIDs is actually referenced and consider pushing familyName all the way down there.